README.md

```diff
@@ -110,6 +110,90 @@ Additionally, CardTextField offers attributes tailored to its purpose (accessibl
 
 ---
 
+### Using the different components of the text field separately
+
+Instead of entering the card number, expiry and CVC in a single text field, it is possible to use single text fields to enter this information separately.
+
```

Comment:
Should this section go at the end? Is it more important than cardio or custom card types?